tests/integrationv2/test_hello_retry_requests.py

```diff
@@ -138,3 +138,55 @@ def test_hrr_with_s2n_as_server(managed_process, cipher, provider, curve, protoc
     assert client_hello_count == 2
     assert server_hello_count == 2
     assert finished_count == 2
+
+# Default Keyshare for TLS v1.3 is x25519 
+TEST_CURVES = ALL_TEST_CURVES[1:] 
```

Comment:
Changes to the `ALL_TEST_CURVES` list will change what gets tested here. I think its ok to make your own list to be explicit about what is actually getting tested.

Reply:
Yes, the intention was to force into hello retry when the default curve is not `x25519`, hence I created a sublist called TEST_CURVES without x25519 in it. Earlier we sent a list of all curves so we wouldn't be retrying for this case.